NAMESPACE

```diff
@@ -173,15 +173,16 @@ importFrom(rlang,quo_name)
 importFrom(rlang,sym)
 importFrom(rlang,syms)
 importFrom(rlang,warn)
-importFrom(stats,as.formula)
 importFrom(stats,setNames)
+importFrom(stringi,stri_wrap)
```

Comment:
Unfortunately no, because stringr::str_wrap collapses white space. We can rethink whether we want to do wrapping before or after alignment, which may help though.